.github/scripts/generate_binary_build_matrix.py

```diff
@@ -207,7 +207,7 @@ def arch_type(arch_version: str) -> str:
     ("cpu", CXX11_ABI): f"pytorch/libtorch-cxx11-builder:cpu-{DEFAULT_TAG}",
 }
 
-FULL_PYTHON_VERSIONS = ["3.9", "3.10", "3.11", "3.12"]
+FULL_PYTHON_VERSIONS = ["3.9", "3.10", "3.11", "3.12", "3.13", "3.13t"]
```

Comment:
Thank you for this change, I believe we had to on generate_wheels_matrix() only since the work for 3.13 was done only for manywheels and not conda